### ISSUE TITLE
[Feature Request] add thresholds to new Range chart 

### ISSUE THREAD

**Issue by t0mZ4** [thingsboard/thingsboard#10037]
We really like the new Range Chart

it is perfect for our temp monitoring applications

but we are missing the important feature of showing Dynamic Threshold lines on the graph, so until this feature is added, we cannot switch to it unfortunately.

![image](https://github.com/thingsboard/thingsboard/assets/31729091/8fc5638d-0b79-4c28-b461-3f467af7f808)

![image](https://github.com/thingsboard/thingsboard/assets/31729091/67b4ca90-038d-4112-a322-80a79b3853d8)

**Comment by ashvayka** [comment]
We will try to fix this into 3.6.3 but most likely it will be part of 3.6.4/3.7 with the new line chart added as well.

**Comment by t0mZ4** [comment]
new line chart!?  this good ol` widget is due a refresh 👍🏾 excited to see the new version.

**Comment by gcorthey** [comment]
This is a very important feature that is also difficulting our migration from AWS/Grafana to TB.

**Comment by ashvayka** [comment]
Yes, we are migrating to the [echarts](https://echarts.apache.org/en/index.html) but it will take some time. Old widgets will become deprecated but we will support them for a few releases. 

Also refactoring the control widgets to be much more user-friendly (basic configuration)

**Comment by t0mZ4** [comment]
very good! please implement their nice Dark Mode as well :) 

![image](https://github.com/thingsboard/thingsboard/assets/31729091/c9d5eea3-866f-4775-a934-1ff9bc753150)

**Comment by ashvayka** [comment]
Hi @t0mZ4 , please attach the screen of the range chart you are using now. I can't understand why you need the thresholds for the range chart. I understand why you need this for the line chart, but it is completely not clear to me why you would like to see it on the range chart. You may attach range chart and add thresholds via any picture editor

**Comment by t0mZ4** [comment]
Hi @ashvayka , sorry for delay, please consider this:


current time series graph:
![image](https://github.com/thingsboard/thingsboard/assets/31729091/a3948587-2a76-4452-a2b1-982f267e46f2)



suggested time range graph:

![image](https://github.com/thingsboard/thingsboard/assets/31729091/e88bdacb-8d14-4879-854d-9c84b6ec0325)


and another example:

![image](https://github.com/thingsboard/thingsboard/assets/31729091/6b21f339-dfd7-4a5e-a3d1-c462714cee79)


vs range graph

![image](https://github.com/thingsboard/thingsboard/assets/31729091/c54c111b-a7bf-45ef-8f64-fa8cbcdd0a50)




and on this regard, i am see that I will be missing the option to add additional data keys to Range Graph, so I cannot create the Alert icon on the graph as I do now. And also, when working in Minus temperatures, I woulr prefer to have control over the display of the color scheme which it seems that I do not?

![image](https://github.com/thingsboard/thingsboard/assets/31729091/c46e1e20-61c9-4695-9270-a0d40a5a6ad4)

hopefully this make sense, let me know if you have other questions

**Comment by mykhailokornieiev** [comment]
Hi. Such a feature was added to Thingsboard in the v3.7 release. 

<img width="893" height="776" alt="Image" src="https://github.com/user-attachments/assets/03ad8d2d-9ed5-4b15-96f1-9e04d71757c5" />
